litert_models/tests/kaggle/tensorflow/mobilenet_v1_test.py

```diff
@@ -12,12 +12,12 @@
 
 
 # https://www.kaggle.com/models/tensorflow/mobilenet-v1/tfLite/0-25-224
-@pytest.mark.xfail(raises=IreeCompileException)
+@pytest.mark.xfail(raises=IreeImportTfLiteException)
```

Comment:
Oh, nice that this actually worked. I was expected we'd need to change https://github.com/iree-org/iree/blob/034bc30519ebe391f3fe2166ff2b148bb8721c12/integrations/tensorflow/python_projects/iree_tflite/iree/tools/tflite/scripts/iree_import_tflite/__main__.py#L14 too.

Ah, since we invoke the tool in a subprocess the NameError there gets rolled into a failure error code here: https://github.com/iree-org/iree-test-suites/blob/f88f598252255147fcc76d092b1478572fbb787b/litert_models/utils.py#L54-L73

If we called the Python API in-process then we'd need different error handling.